Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM ubuntu:xenial as base
+FROM ubuntu:focal as base
 
 # Warning: This file is experimental.
 
 # Install system requirements
 RUN apt-get update && \
     apt-get install -y software-properties-common && \
-    apt-add-repository -y ppa:deadsnakes/ppa && apt-get update && \
```

Comment:
The line above can go away also, although you may need to add `software-properties-common` back to the list of packages below.  See https://github.com/edx/edx-platform/pull/25205/files for the original change.